image_proc/launch/image_proc.launch.py

```diff
@@ -30,36 +30,82 @@
-                    package='image_proc',
-                    node_plugin='image_proc::DebayerNode',
-                    node_name='debayer'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_mono'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_color',
-                    remappings=[
-                        ('image_mono', 'image_color'),
-                        ('image_rect', 'image_rect_color')
-                    ]
-                ),
-            ],
-        ),
-    ])
+    ld = LaunchDescription()
+
+    # Load composable container
+    image_processing = actions.ComposableNodeContainer(
+        node_name="image_proc_container",
+        package='rclcpp_components',
+        node_executable='component_container',
+        composable_node_descriptions=[
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::DebayerNode',
+                node_name='debayer_node',
+            ),
+            # Example of rectifying an image
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::RectifyNode',
+                node_name='rectify_mono_node',
+                # Remap subscribers and publishers
+                remappings=[
+                    # Subscriber remap
+                    ('image', 'image_mono'),
+                    ('camera_info', 'camera_info'),
+                    # Publisher remap
+                    ('image_rect', 'image_rect')
+                ],
+            ),
+            # Example of rectifying an image
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::RectifyNode',
+                node_name='rectify_color_node',
+                # Remap subscribers and publishers
+                remappings=[
+                    # Subscriber remap
+                    ('image', 'image_color'),
+                    ('camera_info', 'camera_info'),
+                    # Publisher remap
+                    ('image_rect', 'image_rect_color')
+                ],
+            )],
+        output='screen'
+    )
+
+    ld.add_action(image_processing)
+
+    return ld
+
+
+def main(argv):
+    ld = generate_launch_description()
+
+    print('Starting introspection of launch description...')
+    print('')
+
+    print(LaunchIntrospector().format_launch_description(ld))
+
+    print('')
+    print('Starting launch of launch description...')
+    print('')
+
+    ls = LaunchService()
+    ls.include_launch_description(get_default_launch_description())
+    ls.include_launch_description(ld)
+
+    return ls.run()
+
+
+if __name__ == '__main__':
+    main(sys.argv)
```

Comment:
You don't need the main function here. With launch files from Dashing forward, you just need to create the `generate_launch_description` function.